spec/performable_mailer_integration_spec.rb

```diff
@@ -22,15 +26,23 @@
         TestMailer.delay.reticulate(@arg)
       end.to change(Delayed::Job, :count).by(1)
       @arg.destroy
-      expect_any_instance_of(Mail::Message).to receive(:deliver).never
+      if ActionMailer::VERSION::STRING >= '4.2.0'
+        expect_any_instance_of(ActionMailer::MessageDelivery).to receive(:deliver_now).never
+      else
+        expect_any_instance_of(Mail::Message).to receive(:deliver).never
+      end
       expect(Delayed::Worker.new.work_off).to eq([1, 0])
     end
     it "ignores deleted models when find doesn't raise an error" do
       expect(TestModel).to receive(:find).with(@arg.id.to_s).and_return(nil)
       expect do
         TestMailer.delay.reticulate(@arg)
       end.to change(Delayed::Job, :count).by(1)
-      expect_any_instance_of(Mail::Message).to receive(:deliver).never
+      if ActionMailer::VERSION::STRING >= '4.2.0'
+        expect_any_instance_of(ActionMailer::MessageDelivery).to receive(:deliver_now).never
+      else
+        expect_any_instance_of(Mail::Message).to receive(:deliver).never
+      end
```

Comment:
I wish there was a neater way to do this, but I don't see how without creating an additional helper or storing these `ActionMailer::MessageDelivery`/`Mail::Message` and `:deliver`/`:deliver_now` values in constants somewhere.